pages/news-and-events/model.ts

```diff
@@ -21,17 +21,29 @@ export const fetchData = async (client: ContentfulClientApi) : Promise<AsyncData
       order: '-fields.publishedDate'
     })
 
+    const heroData = await client.getEntry<HeroData>('4IoMamTLRlN3OpxT1zgnU')
```

Comment:
@matthewtgilbride I wanted to do what I am suggesting here, but TS wasn't allowing it, with the following complaint:
```
var process: NodeJS.Process
Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.ts(2345)
```
```suggestion
    const heroData = await client.getEntry<HeroData>(process.env.ctf_news_and_events_page_id)
```

Reply:
@cameronbaney basically, the type definition for everything on `process.env` is `string | undefined` because it's impossible for TS to know what environment variables might be set.  There are fancier ways of shimming `process.env` to include the things that you expect to be there, but I think that's overkill.

Option 1: `process.env.ctf_news_and_events_page_id as string`
You are basically saying "I know, I know, it's possible this might not be there...but I promise I'll set my env vars when I run this thing".

Option 2 (use a default value): `proccess.env.ctf_news_and_events_page_id ?? ''`.  
That way the type is `string`, and if you accidentally don't set the environment variable, it fetches with empty string as opposed to undefined.

I prefer Option 2 but either is fine.

Reply:
Ah ok, that makes sense. I was thinking that it was something wrong with `getEntry()`'s argument types. I went with option 2.